### ISSUE TITLE
R hangs when running examples

### ISSUE THREAD

**Issue by jeroen** [ropensci/rtweet#706]
Currently the docs site and package checks timeout after 1 hour, because they hang (freeze) in one of the examples:

 - https://github.com/r-universe/ropensci/runs/6219515263?check_suite_focus=true
 - https://github.com/r-universe/ropensci/runs/6219594753?check_suite_focus=true

Maybe it is waiting for user-input?

![Screen Shot 2022-04-29 at 12 34 38 PM](https://user-images.githubusercontent.com/216319/165928901-79d5b94e-37f5-4d5a-8b88-acde33fef4de.png)

**Comment by llrs** [comment]
I recently made the examples runnable, but it shouldn't require user input. On my local machine it goes through without problems (both from devtools::run_examples() and from check pane on RStudio), I will try with the command line to see if it is triggered there.

But after changing to have runnable examples, I had to cancel the action yesterday for similar problems. I saw an error with `tweet_shot()` (connection not closed) but I'm not sure what is the cause of the freeze on Github.
